oppia/oppia.py

```diff
@@ -49,13 +60,31 @@ def resource_string(self, path):
         data = pkg_resources.resource_string(__name__, path)
         return data.decode("utf8")
 
+    def render_template(self, path, context):
+        return self.loader.render_django_template(
+            os.path.join('templates', path),
+            context=Context(context),
+            i18n_service=self.runtime.service(self, "i18n"),
+        )
+
+    def get_translation_content(self):
+        try:
+            return self.resource_string('static/js/translations/{lang}/textjs.js'.format(
+                lang=utils.translation.get_language(),
+            ))
+        except IOError:
+            return self.resource_string('public/js/translations/en/textjs.js')
```

Comment:
We don't have the public folder in the XBlock, should it be created on the fly in the installation? or this default should point to `static`?